SAMLPortal/Middlewares/SetupAssistantMiddleware.cs

```diff
@@ -0,0 +1,38 @@
+using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Provide an 'AssemblyVersion' attribute for this assembly.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)